content/attacks/posts/2022-10-11-Mango-Markets.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2022-10-11
+attacks/posts/categories: Protocol Exploit
+title: "Mango Markets exploited for $116 million"
+---
+
+## Summary
+
+On October 11th, 2022, Mango Markets, a decentralized exchange on Solana was exploited. The hacker manipulated the price oracle for the protocol's MNGO token. The hacker first took out a long MNGO position on Mango. Then the hacker artificially raised the price of MNGO token by advantage of low liquidity on secondaries. The hacker then used the temporary high price of MNGO to take out loans of USDC, various other stable coins, and SOL against unrealized profit on the long MNGO position. 
+
+## Attacker
+
+The Attacker revealed himself in a series of [tweets](https://twitter.com/avi_eisen/status/1581326197241180160) under the pseudonym of Avraham Eisenberg.
+
+
+## Losses
+
+About USD $116 Million worth of crypto assets in terms of USDC, stablecoins and SOL was borrowed and never returned to the platform.
```

Comment:
Funds were recovered by the exploiter, add a source

Reply:
Updated